contracts/src/periphery/cert/interfaces/IEigenDACertTypeBindings.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {EigenDACertTypes} from "src/periphery/cert/EigenDACertTypes.sol";
+
+/// @dev The EigenDA team requires ABIs for EigenDA certificate types.
+///      However, ABIs for types are not generated by the solidity compiler without a function defined.
+///      This interface is simply a workaround for this limitation.
+interface IEigenDACertTypeBindings {
```

Comment:
Should we also have a dummy verification function for V1 so the types can be referenced from the binding?

Reply:
Sure, I can add this.

Reply:
https://github.com/Layr-Labs/eigenda/pull/1518/commits/072c3ada58ccbdbc91fbd6d593389d6e45c4b71f